modules/tinymce/src/themes/silver/main/ts/ui/core/ChoiceControls.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Arr, Fun, Obj, Optional, Singleton, Type } from '@ephox/katamari';
+import { Dimension } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { ContentLanguage } from 'tinymce/core/api/SettingsTypes';
+import { Menu } from 'tinymce/core/api/ui/Ui';
+import { getContentLanguage } from 'tinymce/core/commands/ContentLanguage';
```

Comment:
This is not allowed... you can only import from `api` paths in core. Please move this logic into Silver instead.